src/client/thingtype.cpp

```diff
@@ -196,7 +198,7 @@ void ThingType::unserializeAppearance(uint16_t clientId, ThingCategory category,
         m_market.category = static_cast<ITEM_CATEGORY>(flags.market().category());
         m_market.tradeAs = flags.market().trade_as_object_id();
         m_market.showAs = flags.market().show_as_object_id();
-        m_market.name = flags.market().name();
+        m_market.name = m_name;
```

Comment:
I'm not sure about this, are you sure this will not break compatibility with older versions?

Would be better to check version here.

Reply:
I agree. Do you happen to know in which version apparences.dat started to be used? I believe that if there is any incompatibility it must be from this version, as the others use other means to load the sprites, right? Im without time to test it, but I'll do it as soon as possible.

Reply:
Unfurtunatelly I don't know which version this change was made. But apparences started after 10.98.

Reply:
@pennaor any update on this?

Reply:
> @pennaor any update on this?

Hello friend, well, I don't know how to obtain servers for specific versions, since the canary repository only has releases of version 13.x. I did some research and the market was introduced in version 9.4. This way, it would be good to test with versions 9.4 and 10.98, in addition to 11.x. I'll try with the latest release of TFS, which is 10.98. Any ideas on how to get other versions?

Reply:
That should be enought, if anyone complain about it in the future we just fix it

Reply:
Ok! I'll test 10.98 now and let you know

Reply:
Well, 10.98 still uses .spr/.dat files, but the name of the item and its search in the market works properly. Item details don't show up, but i don't think its because of this changes.
Stash was introduced in version 11.80, so it has not been tested.

Reply:
So should be good to go?

Reply:
For me, yeah!